packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -182,6 +183,14 @@ export const ModalContent: React.FunctionComponent<ModalContentProps> = ({
       aria-labelledby={ariaLabelledbyFormatted()}
       aria-describedby={ariaDescribedby || (hasNoBodyWrapper ? null : descriptorId)}
       {...getOUIAProps(ModalContent.displayName, ouiaId, ouiaSafe)}
+      style={
+        {
+          ...(width && { '--pf-v5-c-modal-box--Width': typeof width !== 'number' ? width : `${width}px` }),
```

Comment:
The check is for the `width` being a `number`, which won't do anything on its own without `px` being added when I was testing locally (I think because it's setting a variable, while Wizard sets the `width` html prop directly).